modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/rel/set/IgniteReduceIntersect.java

```diff
@@ -96,4 +103,5 @@ public <T> T accept(IgniteRelVisitor<T> visitor) {
     public int aggregateFieldsCount() {
         return rowType.getFieldCount() + 2 /* At least two fields required for count aggregation. */;
```

Comment:
Is this calculation still actual?

Reply:
Updated. Replaced 2 with the number of inputs.